Gemfile

```diff
@@ -16,6 +16,7 @@ group :development, :unit_tests do
   gem 'simplecov',               :require => false
   gem 'puppet_facts',            :require => false
   gem 'json',                    :require => false
+  gem 'metadata-json-lint',      :require => false
```

Comment:
I do not know what is meant by 'moduleynced'.  I had to add that line in order to get the tests to pass on my local box.